### DISCUSSION TITLE
Private stencil based component styles not loading

### DISCUSSION THREAD

**Discussion by ashit-darurmath-tfs** [storybookjs/storybook#28702] [Help]
### Summary

Hi everyone, I was trying to implement story book(8.1.11) in a vue(3.2.45) and vite(5.1.5) project and I was consuming stencil components from a private npm repo , stencil core version(4.7.2 )
so the problem is that the component loads but the styles won't load 
here is my main.ts file

```
import type { StorybookConfig } from "@storybook/vue3-vite";
const config: StorybookConfig = {
  stories: ["../src/**/*.mdx", "../src/**/*.stories.@(js|jsx|mjs|ts|tsx)"],
  addons: [
    "@storybook/addon-links",
    "@storybook/addon-essentials",
    "@chromatic-com/storybook",
    "@storybook/addon-interactions",
  ],
  framework: {
    name: "@storybook/vue3-vite",
    options: {},
  },
};
export default config;
```

and here is my preview.ts file

```import type { Preview } from "@storybook/vue3";

const preview: Preview = {
  parameters: {
    controls: {
      matchers: {
        color: /(background|color)$/i,
        date: /Date$/i,
      },
    },
  },
};

export default preview;
```
A answer I found was to import the styles in preview.ts file but the stencil component styles are present in node modules.

**Comment by Marklb** [comment]
I don't have a lot of experience with Vue or Stencil, but I don't think there is enough information to really know.

Are these styles bundled with the component itself or a global stylesheet that is imported separately?

I could be wrong, but to use Stencil in a Vue application, I assume there is some extra Vue configuration. If so, could you show how that is done, especially anything style specific.

Is there a public package that is built similar to your private package and could be used as a replacement to create a small reproduction? If so, https://storybook.new is good for quickly creating a repro.
